### ISSUE TITLE
Optional chaining support isn't working

### ISSUE THREAD

**Issue by bitmanic** [payloadcms/payload#21]
# Bug Report

This project look amazing! I can't wait to check it out. However, I just ran into an issue getting it setup. Here's the skinny:

1. I ran `npx create-payload-app` as instructed in the [Installation](https://payloadcms.com/docs/getting-started/installation) docs.
2. I configured the project. The only thing you might want to know about this part is that I've tried selecting both `javascript` and `typescript` project templates, only to have the save erroneous result.
3. I `cd`ed into the directory and ran `yarn dev`. At this point, `nodemon` starts but the app immediately crashes due to syntax errors.

## Expected Behavior

I expect to be able to run `yarn dev` in a brand new project and have the app run.

## Current Behavior

Here's the output that I see after running `yarn dev`:

```sh
➜  my-project yarn dev
yarn run v1.22.10
$ nodemon
[nodemon] 2.0.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node server.js`
/path/to/my-project/node_modules/payload/dist/fields/traverseFields.js:32
        const hasLocalizedValue = (typeof data?.[field.name] === 'object' && data?.[field.name] !== null)
                                               ^

SyntaxError: Unexpected token '.'
    at Module._compile (internal/modules/cjs/loader.js:892:18)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/path/to/my-project/dist/fields/performFieldOperations.js:8:42)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/path/to/my-project/node_modules/payload/dist/index.js:27:50)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
[nodemon] app crashed - waiting for file changes before starting...
```
It seems clear that the `@babel/plugin-proposal-optional-chaining` plugin isn't being loaded, but I can't figure out why.

## Possible Solution

I'm not sure, but I'll do my best to provide you with whatever further info you need. I appreciate your help!

## Steps to Reproduce
1. Run `npx create-payload-app`.
2. Complete project setup, choosing `javascript` or `typescript` as the project template.
3. `cd` into the project directory and run `yarn dev`. 
4. See error message appear as `nodemon` starts and the app immediately crashes.

## Detailed Description

OS: MacOS Big Sur 11.2
Node: 12.13.0
Payload: 0.1.146

**Comment by denolfe** [comment]
Hey @bitmanic, thanks for the bug report! I was able to recreate this issue. I'll look into getting this fixed 👍 Until then, you can use Node 14.x as a workaround, since optional-chaining is supported natively.

**Comment by bitmanic** [comment]
Thanks for the quick reply, @denolfe! I'll bump my Node version up. ✌️

**Comment by denolfe** [comment]
Fixed with 52a0096d3b8eca47a8afdef42d47117d028b754d
